### PR TITLE
[Event ]Fix incorrect event timestamp

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

We didn't use the correct system clock + always used UTC timestamp, which is bad. It fixes the issue. 

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: src/ray/util/util.h [hunk 1]
```diff
@@ -87,10 +87,9 @@ inline std::string AppendToEachLine(const std::string &str,
   return ss.str();
 }
 
-inline int64_t current_time_s() {
-  std::chrono::milliseconds ms_since_epoch =
-      std::chrono::duration_cast<std::chrono::seconds>(
-          std::chrono::steady_clock::now().time_since_epoch());
+inline int64_t current_sys_time_s() {
```
Comment (rickyyx) [inline src/ray/util/util.h:90]: why is it called ms_xxx?

File: dashboard/state_aggregator.py [hunk 2]
```diff
@@ -593,7 +593,7 @@ async def list_cluster_events(self, *, option: ListApiOptions) -> ListApiRespons
         all_events = await self._client.get_all_cluster_events()
         for _, events in all_events.items():
             for _, event in events.items():
-                event["time"] = str(datetime.utcfromtimestamp(int(event["timestamp"])))
+                event["time"] = str(datetime.fromtimestamp(int(event["timestamp"])))
```
Comment (rickyyx) [inline dashboard/state_aggregator.py:596]: this also changes the schema for cluster event state?

**Review by rickyyx — APPROVED** [review]

File: dashboard/state_aggregator.py (hunk 2 above)
Comment (rkooo567) [reply dashboard/state_aggregator.py:596]: Let me double check. I believe it shouldn't. It just uses the different time zone (previously UTC time zone, now local time zone)

File: src/ray/util/util.h (hunk 1 above)
Comment (rkooo567) [reply src/ray/util/util.h:90]: typo lol. Good catch. Will fix

File: dashboard/state_aggregator.py (hunk 2 above)
Comment (rkooo567) [reply dashboard/state_aggregator.py:596]: Before

```
'time': '2023-04-17 14:12:53'
```

After

```
  'time': '2023-04-17 07:12:27'}]
```

so the same schema

**Pull request merged**